config/locales/en.yml

```
@@ -282,14 +268,13 @@ en:
         </ul>
         <p class="govuk-body">We need to ask you some questions to check if the companies can verify you. We’ll then send you to the website of the company you select.</p>
     will_it_work_for_me:
-      title: Can I be verified?
       heading: About you
       question:
         option_yes: 'Yes'
         above_age_threshold: Are you 20 or over?
         resident_last_12_months: Have you lived in the UK for the last 12 months?
         not_resident_reason:
-          title: Which of these applies to you?
+          sub_heading: Which of these applies to you?
```

```suggestion
          heading: Which of these applies to you?
```
Why `sub_heading` instead of `heading`?